code/modules/awaymissions/mission_code/Cabin.dm

```diff
@@ -123,7 +123,8 @@
 		spawnableAtoms[i] = 1
 
 /datum/mapGeneratorModule/snow/bunnies
-	spawnableAtoms = list(/mob/living/simple_animal/chicken/rabbit/normal = 0.1)
+	//spawnableAtoms = list(/mob/living/simple_animal/chicken/rabbit/normal = 0.1)
+	spawnableAtoms = list(/mob/living/simple_animal/chicken/rabbit = 0.5)
```

Comment:
Ive been asking for weeks whether or not this will work with the damn holidays module, but I am pretty sure this loads before it.